codesign/bin/verify.dart

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io' as io;
+
+import 'package:args/args.dart';
+import 'package:codesign/verify.dart';
+import 'package:logging/logging.dart';
+
+Future<void> main(List<String> args) async {
+  final Logger logger = Logger('root');
+  logger.onRecord.listen((LogRecord record) {
+    io.stdout.writeln(record.message);
+  });
+  final argResults = _parseArgs(args, logger);
+  final VerificationResult result = await VerificationService(
+    binaryPath: argResults['input-file'],
+    logger: logger,
+  ).run();
+  io.exit(result == VerificationResult.codesignedAndNotarized ? 0 : 1);
+}
+
+ArgResults _parseArgs(List<String> args, Logger logger) {
+  final parser = ArgParser()
```

Comment:
Instead of a flag can it read `rest` so instead of:
```
$ dart bin/verify.dart --input-file /Users/fujino/Downloads/darwin_ruby_x64/bin/ruby
```
It's 
```
$ dart bin/verify.dart /Users/fujino/Downloads/darwin_ruby_x64/bin/ruby
```

Reply:
done